doc/integrations/Pytorch/client/package-lock.json

```diff
@@ -1808,7 +1808,6 @@
       "resolved": "https://registry.npmjs.org/@pmmmwh/react-refresh-webpack-plugin/-/react-refresh-webpack-plugin-0.4.3.tgz",
       "integrity": "sha512-br5Qwvh8D2OQqSXpd1g/xqXKnK0r+Jz6qVKBbWmpUcrbGOxUrf39V5oZ1876084CGn18uMdR5uvPqBv9UqtBjQ==",
       "requires": {
-        "ansi-html": "^0.0.7",
```

Comment:
I observed that we are removing `ansi-html` package completely. Can we confirm if the integration codebase works after removing `ansi-html` package. Maybe we need to increase the version and keep `ansi-html` package.

Reply:
Thanks @shailesh-vaidya 

There is no subsequent (Fixed) version of this package.
https://snyk.io/vuln/SNYK-JS-ANSIHTML-1296849 (Please refer remediation section), also there is no subsequent version of ansi-html here->
https://www.npmjs.com/package/ansi-html.

Another possibility is to upgrade webpack-dev-server version from 3.11.1 to 4.3.0, because webpack-dev-server refers to ansi-html-community version
But this change would be even bigger.